### PR TITLE
That's the assignment from Linear Models for Classification

### PR DESCRIPTION
**Pull request by bestfw**
Carried from this website: https://github.com/Nyandwi/machine_learning_complete/blob/main/6_classical_machine_learning_with_scikit-learn/2_linear_models_for_classification.ipynb
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] [local build the book](https://github.com/open-academy/machine-learning/blob/main/CONTRIBUTING.md#build-the-book)
- [ ] review the artifacts from local build
- [x] [local build the slides](https://github.com/open-academy/machine-learning/blob/main/CONTRIBUTING.md#build-the-slides-optional) if appliable
- [ ] Unit test if appliable

# Checklist:

- [x] My code follows the [style guidelines](https://github.com/open-academy/machine-learning/blob/main/STYLE_GUIDE.md) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have applied tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: open-machine-learning-jupyter-book/assignments/ml-fundamentals/build-classification-models.ipynb [hunk 1]
```diff
@@ -5,13 +5,1070 @@
    "cell_type": "markdown",
```
Comment (reveurmichael) [inline open-machine-learning-jupyter-book/assignments/ml-fundamentals/build-classification-models.ipynb:5]: <p>The title should be more specific.</p><p> </p><p>Also, rename this file, it should be more specific as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (reveurmichael) [inline open-machine-learning-jupyter-book/assignments/ml-fundamentals/build-classification-models.ipynb:5]: <pre class="ql-syntax" spellcheck="false">Line #4.    # mobile_test.head(2)
remove mobile_test.head(2), or put it in a new cell
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (reveurmichael) [inline open-machine-learning-jupyter-book/assignments/ml-fundamentals/build-classification-models.ipynb:5]: <p>remove</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (reveurmichael) [inline open-machine-learning-jupyter-book/assignments/ml-fundamentals/build-classification-models.ipynb:5]: <p>soo -&gt; so</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Comment (reveurmichael) [inline open-machine-learning-jupyter-book/assignments/ml-fundamentals/build-classification-models.ipynb:5]: <p>It's not from Microsoft.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='96'/>

File: open-machine-learning-jupyter-book/assignments/ml-fundamentals/Build-classification-models-Predict-the-price-range.ipynb [hunk 2]
```diff
@@ -0,0 +1,1096 @@
+{
```
Comment (reveurmichael) [inline open-machine-learning-jupyter-book/assignments/ml-fundamentals/Build-classification-models-Predict-the-price-range.ipynb:1]: <p>line 9-10, two empty lines</p><p> </p><p>should be 0 or 1</p><p> </p><p>line 16 add empty line</p><p> </p><p>line 15: remove To</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (reveurmichael) [inline open-machine-learning-jupyter-book/assignments/ml-fundamentals/Build-classification-models-Predict-the-price-range.ipynb:1]: <p>No first empty line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Comment (reveurmichael) [inline open-machine-learning-jupyter-book/assignments/ml-fundamentals/Build-classification-models-Predict-the-price-range.ipynb:1]: <p>no first empty line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Comment (reveurmichael) [inline open-machine-learning-jupyter-book/assignments/ml-fundamentals/Build-classification-models-Predict-the-price-range.ipynb:1]: <p>a space before <a href="https://github.com/Nyandwi/machine_learning_complete" target="_blank">machine_learning_complete</a>. A dot at the end.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='96'/>

**Comment by reveurmichael** [comment]
no Upper Case in file title, all lower case

File: open-machine-learning-jupyter-book/assignments/ml-fundamentals/Build-classification-models-Predict-the-price-range.ipynb (hunk 2 above)
Comment (bestfw) [reply open-machine-learning-jupyter-book/assignments/ml-fundamentals/Build-classification-models-Predict-the-price-range.ipynb:1]: <p>I don't understand this comment very well, in the code ,it is:</p><p> </p><p>&lt;a name='5'&gt;&lt;/a&gt;</p><p> </p><p><strong>## 5. Creating and Training a Logistic Regression Model</strong></p><p> </p><p>We will start by importing <code>LogisticRegression</code> models available in Sklearn linear models and then proceed with model training. </p><p> </p><p>So I think there is empty line now.</p><p> </p>

**Review by reveurmichael — APPROVED** [review]
OK

**Pull request merged**